src/components/ActionList/components/Item/Item.tsx

```diff
@@ -81,12 +82,12 @@ export function Item({
   const textMarkup = <div className={styles.Text}>{contentMarkup}</div>;
 
   const contentElement = (
-    <div className={styles.Content}>
+    <span className={styles.Content}>
```

Comment:
You can't have block level elements inside of inline elements. https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#conceptual_differences

Reply:
That’s the reason we changed it to a span.

Reply:
Oh, but `textMarkup` from above is still a `div`?

Reply:
I went in and fix all nested divs for this